src/lib/client.ts

```diff
@@ -7,16 +7,19 @@ export class UXDClient {
     constructor({
         provider,
         controllerAddress,
-        uxdTokenAddress
+        uxdTokenAddress,
+        market
     }: {
         provider: providers.JsonRpcProvider;
         controllerAddress: string;
         uxdTokenAddress: string;
```

Comment:
`redeemable` ?

Reply:
rename from `uxdTokenAddress` to `redeemable` to follow same naming convention as on Solana

Reply:
Why do we want to follow the naming convention on Solana?

Reply:
Change to `redeemable`